ggml/src/ggml.c

```diff
@@ -18590,9 +18644,285 @@ static int ggml_get_n_tasks(struct ggml_tensor * node, int n_threads) {
     return n_tasks;
 }
 
-struct ggml_cplan ggml_graph_plan(const struct ggml_cgraph * cgraph, int n_threads) {
+static thread_ret_t ggml_graph_compute_secondary_thread(void* data);
+
+enum {
+    SCHED_PRIO_NORMAL,
+    SCHED_PRIO_MEDIUM,
+    SCHED_PRIO_HIGH,
+    SCHED_PRIO_REALTIME
+};
+
+#if defined(_WIN32)
+#include "windows.h"
+
+// TODO: support > 64 CPUs
+static bool __thread_affinity(bool * mask) {
```

Comment:
Identifiers that start with two underscores are reserved for the compiler.

Reply:
Linux kernel habits  :) __something usual means special calls not to be called from wrong context (must be under lock, etc).
Will rename to ggml_thread_apply_affinity() tomorrow.